packages/website/docs/creating-a-renderer.md

```diff
@@ -3,17 +3,17 @@ id: creating-a-renderer
 title: Creating a Renderer
 ---
 
-Learn how to build a autocomplete UI using React.
+Learn how to build an autocomplete UI using React.
 
-The `@algolia/autocomplete-core` package provides all the primitives to build an autocomplete experience, but you remain in charge of the UI output. This page will teach you how to leverage all the autocomplete capacities to build an accessible autocomplete, both for desktop and mobile.
+The [`autocomplete-js`](autocomplete-js) package includes everything you need to render a vanilla JavaScript autocomplete experience. If you want to build a custom renderer, say in [React](https://reactjs.org/docs/getting-started.html) or another front-end framework, the [`autocomplete-core`](createAutocomplete) package provides all the primitives to build it.
```

Comment:
Since the creation of this guide, Autocomplete JS has become framework-agnostic. You can now plug your own framework. The goal of this guide has shifted to building a UI that is different from the provided package (e.g., the new Algolia documentation search).

```suggestion
The [`autocomplete-js`](autocomplete-js) package includes everything you need to render a JavaScript autocomplete experience that you can bind to [your own framework](autocomplete-js#renderer). If you want to build a custom UI that differs from the `autocomplete-js` output (being in [React](https://reactjs.org/docs/getting-started.html) or another front-end framework), the [`autocomplete-core`](createAutocomplete) package provides all the primitives to build it.
```